### PR TITLE
Working on new model infra

### PR DESCRIPTION
**Pull request by kingst**

### CODE REVIEW

File: base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java [hunk 1]
```diff
@@ -168,6 +226,30 @@ public void run() {
...
+        final RunArguments args = getNextImage();
+
+        Bitmap bm;
+        if (args.mFrameBytes != null) {
+            bm = getBitmap(args.mFrameBytes, args.mWidth, args.mHeight, args.mFormat,
+                    args.mSensorOrientation, args.mRoiCenterYRatio);
+        } else if (args.mBitmap != null) {
+            bm = args.mBitmap;
+        } else {
+            bm = Bitmap.createBitmap(480, 302, Bitmap.Config.ARGB_8888);
```
Comment (dxaen) [inline base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java]: We don't need to do this for object detection or do we?

File: base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java [hunk 2]
```diff
@@ -126,25 +178,31 @@ private synchronized RunArguments getNextImage() {
...
+                try {
+                    if (args.mObjectListener != null) {
+                        /*
+                        if (hadUnrecoverableException) {
+                            args.mScanListener.onFatalError();
+                        } else {
+                            args.mScanListener.onPrediction(number, ocr.expiry, bitmap, ocr.digitBoxes,
+                                    ocr.expiryBox);
+                        }*/
+                        args.mObjectListener.onPrediction(bitmap);
```
Comment (dxaen) [inline base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java]: We will need to add a few fields here.

**Comment by dxaen** [comment]
It all looks good to me. #shipit

File: base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java (hunk 1 above)
Comment (kingst) [reply base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java]: If we can skip it then that's better, but the current example is using exactly this (as far as I can tell)

File: base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java (hunk 2 above)
Comment (kingst) [reply base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java]: yeah, definitely, I just wasn't sure what these were :)

**Pull request merged**